src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountWeek = 50;
+  const discountThreeDays = 20;
+  const totalPrice = days * price;
+
+  if (days >= 7) {
+    return totalPrice - discountWeek;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalPrice - discountThreeDays;
+  }
+
+  if (days < 3) {
+    return totalPrice;
+  }
```

Comment:
functions should have at least 1 consistent return, you also checked all the conditions before, so this if is redundant
```suggestion
  return totalPrice;
```